### DISCUSSION TITLE
NUnit colors for states

### DISCUSSION THREAD

**Discussion by yetibrain** [nunit/nunit#3902] [Q&A] [answered]
Hi all,

is there any color scheme for NUnit result states? I mean usually for failed one would choose red, for passed green. Possibly yellow for ignored and possibly purple for inconclusive. Is there any standard? Thanks for any hint and/or link.

Peter

**Comment by mikkelbu** [comment, marked as answer]
Hi @yetibrain 
As far as I know there is no standard, and different runners can choose their own colour scheme, but the console follows more or less your idea. I cannot remember if it is the same for the gui.

**Reply by yetibrain** [reply]
Thanks for your reply.
